scripts/superdesk-authoring/authoring.js

```diff
@@ -3206,7 +3213,18 @@
          */
         this.edit = function(item, action) {
             if (item) {
```

Comment:
Is there ever a case when no `item` is provided for editing? If no, then the condition is not needed imo.